src/test/scala/wdl4s/expression/ValueEvaluatorSpec.scala

```diff
@@ -25,6 +25,7 @@ class ValueEvaluatorSpec extends FlatSpec with Matchers {
       WdlString("c") -> WdlInteger(2)
     ))
     case "o" => WdlObject(Map("key1" -> WdlString("value1"), "key2" -> WdlInteger(9)))
+    case "myPair" => WdlPair(WdlInteger(3), WdlString("hello"))
```

Comment:
This was actually already there, only now it's evaluated as a tuple1 rather than ignored for us by the parser (e.g. line 177)